js/client-mainmenu.js

```diff
@@ -1101,30 +1100,94 @@
 			} else if (teams.length > 27) {
 				bufBoundary = Math.ceil(teams.length / 2);
 			}
-
+			this.folderNotExpanded = data.folderNotExpanded || [];
+			this.folderToggleOn = data.folderToggleOn;
+			var folders = {};
 			this.team = data.team;
 			this.format = data.format;
 			this.room = data.room;
-
+			this.moreTeamsTemp = data.moreTeamsTemp || false;
```

Comment:
`moreTeamsTemp` is a very poor name. this is a boolean, so a name beginning with `is` or `should` would probably be a good place to start.